microsite/data/plugins/tech-updates.yaml

```diff
@@ -0,0 +1,10 @@
+---
+title: Tech Updates
+author: Infosys
+authorUrl: https://www.infosys.com
+category: Utility
+description: A tech news updates is to keep users informed about the latest updates.
+documentation: https://github.com/AmbrishRamachandiran/tech-updates/blob/main/README.md

```

Comment:
Few recommendations on your `README`:

- There is a link that's broken, seems the file is missing
- Would be great to have pictures that show a bit more what the news would look like
- The install instructions are all in one large code block which makes it hard to read